azure-pipelines.yml

```diff
@@ -65,7 +65,14 @@ steps:
     Contents: '$(Build.BuildId).zip'
     TargetFolder: '$(Build.ArtifactStagingDirectory)\ArtifactsToBePublished'
 
-- task: PublishBuildArtifacts@1
-  displayName: 'Publish Artifact: drop'
+- task: AzureWebApp@1
+  displayName: 'Deploy Azure App Service'
   inputs:
-    PathtoPublish: '$(Build.ArtifactStagingDirectory)\ArtifactsToBePublished'
+    azureSubscription: 'hazzat-api-ppe - Azure'
+    appType: 'webApp'
+    appName: 'hazzat-api-ppe'
+    deployToSlotOrASE: true
+    resourceGroupName: 'hazzat-api-ppe-rg'
+    slotName: 'production'
```

Comment:
I don't have slots in the ppe environment.  It's not available in this sku.  Will this work still?

Reply:
[Yes](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-rm-web-app-deployment?view=azure-devops), the flag `deployToSlotOrASE` works for either or as long as the `slotName` is set to `production`